news/2024/2024-10-09-resurrection-cup-2024-results.md

```diff
@@ -0,0 +1,415 @@
+---
+layout: post
+title: Resurrection Cup 2024 Results
+date: 2024-10-09 19:00:00 +0000
+---
+
+This year's Resurrection Cup has come to an end... read on for the results and reflections from the people who made it happen!
+
+![](/wiki/shared/news/2024-05-12-resurrection-cup-2024-registrations-now-open/banner.jpg)
+
+Starting with [registrations back in May 11](https://osu.ppy.sh/home/news/2024-05-12-resurrection-cup-2024-registration-now-open) and ending with a grand final showdown on July 28, the Resurrection Cup has ended another year with a bang.
+
+## Results
+
+![](/wiki/shared/news/2024-10-09-resurrection-cup-2024-results/podium.jpg)
+
+## Highlights
+
+In the Quarterfinals, the legendary [verto](https://osu.ppy.sh/users/2015300) made his appearance as a commentator. Known not for his tournament commentary, but for the iconic "Greetings!" that every osu! boomer instantly recognises from their [osu! YouTube content](https://www.youtube.com/@vertoHUN). Together with veteran [SadShiba](https://osu.ppy.sh/users/10747626), they offered thrilling commentary to the delight of viewers. Not only that, but the intense match, moving from breakpoint to breakpoint and stretching well over an hour, is an absolute must watch. 
+
+*Germany owc B*, not favoured to win it, put up an incredible fight and almost managed to cause an upset. However, despite their determined efforts coming back from a 4-2 deficit - @ [57:30](https://youtu.be/2l4KS5feTA8?t=3450) - *Germany owc B* ultimately couldn't keep up and fell just short. Right before a tiebreaker could be reached, they lost to *swaglords*' pick of [Drastic Dramatic](https://osu.ppy.sh/beatmapsets/2206803#osu/4672803) (FM4) - @ [1:17:45](https://youtu.be/2l4KS5feTA8?t=4665). That being said, their battle was nothing short of exciting and a true testament to osu! players' competitive spirit.
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/2l4KS5feTA8" frameborder="0" allowfullscreen></iframe>
+</div>
+
+In the Finals, top-seeded *kotomiler* faced off against the second seed *fresh off the boat* in a highly anticipated matchup. The intense game saw both sides trading wins on their own picks until the last two picks, which both turned into breakpoints - @ [1:25:40](https://youtu.be/R0FSFEV_uLA?t=5140). The match culminated in a tiebreaker, where ::{ flag=KR }:: [FlyingTuna](https://osu.ppy.sh/users/9224078) gave an incredible performance by S-ranking Shokiii’s original Finals Tiebreaker song, [Eterna](https://osu.ppy.sh/beatmapsets/2215097#osu/4693364). Despite finding a sliderbreak in the final kiai section, this performance pulled team *fresh of the boat* across the finish, securing them the victory in the match - @ [1:38:08](https://youtu.be/R0FSFEV_uLA?t=5888).
```

Comment:
reads weird to have `picks` repeated in quick succession, also i feel like the last sentence is better off without a timestamp at the end
```suggestion
In the Finals, top-seeded *kotomiler* faced off against the second seed *fresh off the boat* in a highly anticipated match-up. The intense game saw both sides trading wins on their own picks until the last two maps, which both turned into breakpoints @ [1:25:40](https://youtu.be/R0FSFEV_uLA?t=5140). The match culminated in a tiebreaker, where ::{ flag=KR }:: [FlyingTuna](https://osu.ppy.sh/users/9224078) gave an incredible performance by S-ranking Shokiii’s original Finals Tiebreaker song, [Eterna](https://osu.ppy.sh/beatmapsets/2215097#osu/4693364), @ [1:38:08](https://youtu.be/R0FSFEV_uLA?t=5888). Despite finding a sliderbreak in the final kiai section, this performance pulled team *fresh of the boat* across the finish, securing them the victory in the match.
```

Reply:
matchup is correct though no? Agree with the other changes.

Reply:
yea you're right ive amended my review, about the `picks` vs. `maps` im not overly critical of `picks` staying twice

Reply:
i amended the suggestions once again to completely reword that sentence as discussed on [Discord](https://discord.com/channels/188630481301012481/1233349822359273504/1302613245038825522)

Reply:
Resolved following the discussion on [Discord](https://discord.com/channels/188630481301012481/1233349822359273504/1302629234409472031)